bin/publish_website

```diff
@@ -18,6 +20,6 @@ docker run --rm \
   -e AWS_ACCESS_KEY_ID \
   -e AWS_SECRET_ACCESS_KEY \
   mesosphere/aws-cli \
-    s3 sync --delete /_site s3://${S3_BUCKET}
+    s3 sync --delete "/_site s3://${S3_BUCKET}"
```

Comment:
I don't know how to test this - but I worry about this failing when it gets merged to master.

Reply:
i am like 99% confident that change won’t break anything, so if it was working before it will still be working.  the previous would fail if the S3_BUCKET name has spaces in it, which is probably not allowed anyway, but these rules exists so you don’t you think about that stuff.